test/swirl.test.ts

```diff
@@ -0,0 +1,246 @@
+import { TransactionType } from '@msafe/iota-utils';
+import { ChangeBaseRewardFeeIntention, ChangeBaseRewardFeeIntentionData } from '@/apps/swirl/intentions/changeBaseRewardFee';
+import { ChangeMinStakeIntention, ChangeMinStakeIntentionData } from '@/apps/swirl/intentions/changeMinStake';
+import { StakeIntention, StakeIntentionData } from '@/apps/swirl/intentions/stake';
+import { UnstakeIntention, UnstakeIntentionData } from '@/apps/swirl/intentions/unstake';
+import { UpdateRewardsRevertIntention, UpdateRewardsRevertIntentionData } from '@/apps/swirl/intentions/updateRewardsRevert';
+import {
+  UpdateRewardsThresholdIntention,
+  UpdateRewardsThresholdIntentionData,
+} from '@/apps/swirl/intentions/updateRewardsThreshold';
+import { TransactionSubType } from '@/apps/swirl/types';
+import { appHelpers } from '@/index';
+
+import { Account, Client } from './config';
+
+describe('Swirl Wallet', () => {
+  const appHelper = appHelpers.getAppHelper('Swirl');
+
+  it('Should get correct app helper', () => {
+    expect(appHelper.application).toBe('Swirl');
+  });
+
+  describe('Build/Serialize', () => {
+    it('Change Base Reward Fee transaction build', async () => {
+      const intentionData: ChangeBaseRewardFeeIntentionData = {
+        owner_cap: '',
```

Comment:
I don't know really what should be here for typedArguments? Can you explain please?